internal_test_data/enospc-30mins_new.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 5
```

Comment:
think with Roy about the name

Reply:
These yamls shouldn't be here.
If this is a sample yaml, please call it that way.
Also, no need for so many samples, we should give as sample the "thinnest" possible way or our best practice so people will follow only this sample.

Reply:
Change the directory name to "test_configurations"

Reply:
IIUC, these are internal tests for SCT to test the new configs.
In that case, let's give a meaningful names for the yamls instead of confusing names.
For example: internal_test_data/use-params-only-from-yaml.yaml